cmd/antrea-agent/options.go

```diff
@@ -534,6 +544,16 @@ func (o *Options) validateK8sNodeOptions() error {
 		o.config.TunnelType != ovsconfig.GRETunnel && o.config.TunnelType != ovsconfig.STTTunnel {
 		return fmt.Errorf("tunnel type %s is invalid", o.config.TunnelType)
 	}
+	if !utilconfig.IsValidPort(o.config.ClusterMembershipPort) {
+		return fmt.Errorf("invalid clusterPort %d", o.config.ClusterMembershipPort)
```

Comment:
do you think `IsValidPort` should return an error instead, would that simplify usage slightly:
```go
if err := utilconfig.ValidatePort(o.config.ClusterMembershipPort); err != nil {
        return fmt.Errorf("invalid clusterMembershipPort: %w", err)
}
```

`ValidatePort` could return an error which includes the user-provided port value, so we don't need to repeat `o.config.ClusterMembershipPort`

Reply:
Thanks. Updated to return an error to simplify the usage.